CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 * Added HelloWorldSlim template option for *--template* flag in **demisto-sdk init** command.
 * Fixed an issue where the HelloWorldSlim template in **demisto-sdk init** command had an integration id that was conflicting with HelloWorld integration id.
 * Updated the SDK to use demisto-py 3.1.6, allowing use of a proxy with an environment variable.
+* Removed a false warning about configuring the `GITLAB_TOKEN` environment variable when it's not needed.
```

Comment:
please merge from master, list seems outdated

Reply:
moved to unreleased